<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN-L2504" ignored="0" name="Financial Fees" tests="34" failures="27" timestamp="18 May 2021 19:14:09 GMT" time="2545.057" errors="0">
  <testcase name="testGetNotificationsForFinancialFees" time="86.545" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="84.378" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees"/>
  <testcase name="testGetNotificationsForFinancialFees" time="100.385" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="103.615" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees"/>
  <testcase name="testGetNotificationsForFinancialFees" time="85.043" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="73.583" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[java.lang.AssertionError: Expected  2 insights triggered But found Empty {}
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:346)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="86.362" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees"/>
  <testcase name="testGetNotificationsForFinancialFees" time="85.616" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="85.362" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="66.713" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="66.824" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="65.221" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="69.052" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees"/>
  <testcase name="testGetNotificationsForFinancialFees" time="71.126" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="69.689" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="69.379" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="67.714" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="67.659" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="67.949" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="73.234" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="75.37" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="72.052" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="71.892" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees"/>
  <testcase name="testGetNotificationsForFinancialFees" time="87.527" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="72.883" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="71.219" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="79.636" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="79.333" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="71.746" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees"/>
  <testcase name="testGetNotificationsForFinancialFees" time="72.765" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees"/>
  <testcase name="testGetNotificationsForFinancialFees" time="74.098" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="70.402" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="74.451" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:326)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
  <testcase name="testGetNotificationsForFinancialFees" time="4.962" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="0">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 0
	at com.yodlee.insights.yodleeApi.utils.InsightsGenerics.getTransactionIds(InsightsGenerics.java:273)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForFinancialFees.testGetNotificationsForFinancialFees(TestGetInsightsForFinancialFees.java:277)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForFinancialFees -->
</testsuite> <!-- Financial Fees -->
